src/app/components/mnoe-sub-tenants-list/mnoe-sub-tenants-list.html

```diff
@@ -0,0 +1,48 @@
+<mno-widget icon="fa-wrench" heading="{{$ctrl.subTenant.widgetTitle}}" is-loading="subTenant.loading">
+  <mno-widget-header translate>
+    &nbsp;mnoe_admin_panel.dashboard.sub_tenants.widget.list.title
+  </mno-widget-header>
+  <mno-widget-body classes="large no-padding">
+    <div>
+      <table st-pipe="$ctrl.callServer" st-table="$ctrl.listOfSubTenant" class="table table-striped" >
+        <thead>
+          <tr>
+            <th st-sort="name" class="col-lg-2" translate>mnoe_admin_panel.dashboard.sub_tenants.widget.list.table.name</th>
+            <th st-sort="created_at" class="col-lg-2" translate>mnoe_admin_panel.dashboard.sub_tenants.widget.list.table.created_at</th>
+            <th st-sort="update_at" class="col-lg-2" translate>mnoe_admin_panel.dashboard.sub_tenants.widget.list.table.updated_at</th>
+        	</tr>
```

Comment:
I think it is a bug of github diff viewer: 
![image](https://user-images.githubusercontent.com/4570217/29624363-c87ed4e8-8820-11e7-8446-9cfac1ebcc5b.png)